src/Validation.ContentScan.Core/CheckContentScanData.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NuGet.Jobs.Validation.ContentScan
+{
+    public class CheckContentScanData
+    {
+        public static CheckContentScanData NewStartContentScanData(
+           Guid validationTrackingId,
```

Comment:
These should be the `validationStepId`.

```suggestion
           Guid valiadtionSetId,
```

Please see my comment here: https://github.com/NuGet/NuGet.Jobs/pull/970/files#discussion_r611773364

Old name, don't use | New name, please use
-- | --
Validation tracking ID | Validation set ID
Validation ID | Validation step ID

You passed in the `validationStepId` here: https://github.com/NuGet/NuGet.Jobs/pull/970/files#diff-5277c5c017c60bf6ce02d7872f34805c4ae9c08984ab7e6f48f6d877b7d7296bR79